src/oaklib/implementations/semsimian/semsimian_implementation.py

```diff
@@ -137,19 +137,16 @@ def all_by_all_pairwise_similarity(
         objects = list(objects)
         logging.info(f"Calculating all-by-all pairwise similarity for {len(objects)} objects")
         all_results = self.semsimian.all_by_all_pairwise_similarity(
-            set(subjects), set(objects), set(predicates) if predicates else None
+            set(subjects),
+            set(objects),
+            min_jaccard_similarity,
```

Comment:
positional arguments make me very uneasy. What happens when we add more methods, we want to add filters for these. We end up with 10 arguments with high potential for error in aligning arguments...

If rust does not support named arguments I think we need to move towards a config object. This can be done later but is important

Reply:
Rust does support named arguments. Making the change.